tests/console/http_srv.pm

```diff
@@ -16,11 +16,16 @@ use warnings;
 use base "consoletest";
 use testapi;
 use utils;
+use version_utils 'is_sle';
 
 sub run {
     select_console 'root-console';
     # Log space available before installation, see poo#19834
     script_run("df -h > /dev/$serialdev", 0);
+    # In upgrade scenario we have service_check module and apache2 already installed
+    if ((get_var('UPGRADE') == 1) || get_var('MIGRATION_METHOD')) && is_sle {
```

Comment:
Seems you are missing some brackets. The part I don't get, why do we want to remove `apache` and install it again? I guess we should rather flip the condition and install apache in case it's not `UPGRADE` or `MIGRATION_METHOD`. WDYT?

Reply:
For the brackets issue i will check it and correct it.
Since this http_serv.pm not design for migration , it will install apache and start check whether apache is enable or not. In migration scenario we will face issue since before migration the apache already installed and enabled, so after migration the apache already installed and enabled.  
That's why we need uninstall it before start check service status.

Reply:
To be honest I don't get why you need this test module at all in the migration scenario, as you've already tested that service is up and running. If you need to test clean apache installation on top, then maybe we should remove apache in the module where you check the migrated service.

Reply:
Why this service triggered is caused by an workround/hack on s390, other platform can not schedule this module.  
In products/sle/main.pm:
if (is_sle('15+')) {
     if (check_var('ARCH', 's390x') and get_var('DESKTOP', '') =~ /gnome|minimalx/) {
         diag 'BUG: bsc#1058071 - No VNC server available in SUT, disabling X11 tests. Re-enable after bug is fixed';
         set_var('DESKTOP', 'textmode');   <====== this set to textmode !!!!!
     }
 }
Since 390 DESKTOP set to textmode, so http_srv is scheduled within main_common.pm
In main_common.pm:
    if (!get_var("NOINSTALL") && !get_var("LIVETEST") && (check_var("DESKTOP", "textmode"))) {
         loadtest "console/mysql_srv";
         # disable these tests of server packages for SLED (poo#36436)
         load_console_server_tests() unless is_desktop;   <===== load_console_server_tests will call http_srv.pm
     }

Reply:
Martin just give me a ticket for fix this https://progress.opensuse.org/issues/50285
So if above ticket fixed, then we do not need this PR.
But i think we need this kind of function check even after migration, then we need find solution for this.